x/inter-tx/client/cli/flags.go

```diff
@@ -5,16 +5,18 @@ import (
 )
 
 const (
-	FlagConnectionId             = "connection-id"
-	FlagCounterpartyConnectionId = "counterparty-connection-id"
+	// The connection end identifier on the controller chain
+	FlagConnectionID = "connection-id"
+	// The connection end identifier on the host chain
+	FlagCounterpartyConnectionID = "counterparty-connection-id"
```

Comment:
should we just remove this and refactor to get the counterparty connection ID from the connection ID?

Reply:
I can look at doing that in a follow up PR for sure.